tests/test_base_script.py

```diff
@@ -235,46 +250,73 @@ def test_non_configurable_script_get_schema(self):
 
             self.assertFalse(script.done_task.done())
 
+    async def test_next_supplemented_group_id(self):
+        async with salobj.TestScript(index=self.index) as script:
+            await self.configure_and_check(script)
+            group_id = script.group_id
+            for i in range(5):
+                desired_supplemented_id = group_id + "-" + str(i+1)
```

Comment:
Consider:
```python
desired_supplemented_id = f"{group_id}-{i+1}"
```

Reply:
Will do. I was on the fence but I agree.